src/core/UserIconGroup/UserIconGroup.tsx

```diff
@@ -0,0 +1,133 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import { useTheme } from '../utils/hooks/useTheme';
+import cx from 'classnames';
+import { UserIcon } from '../UserIcon';
+export type UserIconGroupType = 'default' | 'animated';
+
+const defaultLength = 5;
+const maxLength = 9;
+
+export type UserIconGroupProps = {
+  /**
+   * UserIconGroup group length
+   * @default 5
+   */
+  length?: number;
```

Comment:
This prop name could be confusing. 🤔

Maybe `maxLength` or `maxIcons`?

Reply:
I chose maxIcons :)